File: imobil-project/bin/Casa.dart
import 'imobil.dart';
import 'Proprietar.dart';
import 'Poza.dart';
class Casa extends Imobil
{
  int nr_camere;
  int etaje;
  String curte;
  Casa(Proprietar proprietar, String locatie, int dimensiune, Poza poze, int nr_camere, int etaje,String curte) : super(proprietar, locatie, dimensiune, poze)
  {
    this.nr_camere = nr_camere;
    this.etaje = etaje;
    this.curte = curte;

  }
  int get Numar_De_Camere => nr_camere;
  set Numar_De_Camere(int value) => nr_camere=value;

  int get Etaje => etaje;
  set Etaje(int value) => etaje=value;

  String get Curte =>curte;
  set Curte(String value) =>curte=value;

  @override
  String toString() {
    StringBuffer sb = new StringBuffer();
    sb.writeln('~~CASA~~');
    sb.writeln('PROPRIETAR: ' +proprietar.toString());
    sb.writeln('LOCATIE: '+locatie.toString());
    sb.writeln('DIMENSIUNE: '+dimensiune.toString());
    sb.writeln('POZE: '+poze.toString());
    sb.writeln('NUMAR DE CAMERE : '+nr_camere.toString());
    sb.writeln('ETAJe : '+etaje.toString());
    sb.writeln('CURTE '+curte.toString());
    return sb.toString();

  }
}

File: Imobil C#/Imobil/Casa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Imobil
{
    class Casa: Imobil
    {
        public int nr_camere;
        public int etaje;
        public string curte;
        public Casa(Proprietar proprietar, string locatie, int dimensiune, Poza poze, int nr_camere, int etaje,string curte) : base(proprietar, locatie, dimensiune, poze)
        {
            this.nr_camere = nr_camere;
            this.etaje = etaje;
            this.curte = curte;
        }
        public int Numar_De_Camere
        {
            get
            {
                return nr_camere;
            }
            set
            {
                nr_camere = value;
            }
        }
        public int Etaje
        {
            get
            {
                return etaje;
            }
            set
            {
                etaje = value;
            }
        }
        public string Curte
        {
            get
            {
                return curte;
            }
            set
            {
                curte = value;
            }
        }
        public override string ToString()
        {
            StringBuilder sb = new StringBuilder();
            sb.AppendLine("~~CASA~~");
            sb.AppendLine($"PROPRIETAR: {proprietar}");
            sb.AppendLine($"LOCATIE: {locatie}");
            sb.AppendLine($"DIMENSIUNE: {dimensiune}");
            sb.AppendLine($"POZE: {poze}");
            sb.AppendLine($"NUMAR DE CAMERE : {nr_camere}");
            sb.AppendLine($"ETAJE : {etaje}");
            sb.AppendLine($"CURTE: {curte}");
            return sb.ToString();
        }
    }
}
